.travis.yml

```diff
@@ -39,9 +39,15 @@ compiler:
 env:
   matrix:
   - Q_OR_C_MAKE=cmake
-    QT_VERSION=59 # actually 5.9
+    QT_VERSION=59
+    WITH_FONTS=NO # actually 5.9
```

Comment:
Is this just for the PR or intended to go into the repository?

Reply:
Just for testing - it expands the CI build matrix more than I would like (I spent several hours late last night/early this morning watching the [Travis MacOS](https://www.traviscistatus.com/) jobs backlog declining slightly - WTF happened at 04:00 UTC - and I do not want to load that more than I have to). Will revert now the code compiles on everything.